proxy/src/proxy.rs

```diff
@@ -324,20 +324,42 @@ async fn connect_to_compute(
     extra: &console::ConsoleReqExtra<'_>,
     creds: &auth::BackendType<'_, auth::ClientCredentials<'_>>,
 ) -> Result<PostgresConnection, compute::ConnectionError> {
-    let mut num_retries: usize = NUM_RETRIES_WAKE_COMPUTE;
+    let mut num_retries: usize = 0;
```

Comment:
If my reading is correct you also need to change `num_retries -= 1` to `num_retries += 1` and retries are not happening in this code because -1 causes `num_retries` to overflow to smth that is greater than 1

Reply:
Good spot. I've reverted this back to the old style anyway